magneticow/magneticow/magneticow.py

```diff
@@ -67,7 +67,17 @@ def decorated(*args, **kwargs):
 @app.route("/")
 @requires_auth
 def home_page():
-    return flask.render_template("homepage.html")
+    magneticod_db = get_magneticod_db()
+    context = {}
+
+    with magneticod_db:
+        cur = magneticod_db.execute(
+            "SELECT"
+            "    count(id) "
+            "FROM torrents ;")
+        context["counter"] = cur.fetchall()
```

Comment:
I think it would be better if you used a different name, maybe `n_torrents` (**n**umber of **torrents**)? Because *counter* in itself is not very clear (counter of what?).

Also

    context["counter"] = cur.fetchall()

should become

    context["NEW NAME"] = cur.fetchone()[0]

for clarity. =)